bundles/config/org.eclipse.smarthome.config.dispatch/META-INF/MANIFEST.MF

```diff
@@ -5,12 +5,13 @@ Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-SymbolicName: org.eclipse.smarthome.config.dispatch
 Bundle-Vendor: Eclipse.org/SmartHome
 Bundle-Version: 0.9.0.qualifier
-Import-Package: 
- org.apache.commons.io;version="2.2.0",
- org.apache.commons.lang;version="2.6.0",
- org.eclipse.jdt.annotation;resolution:=optional,
- org.eclipse.smarthome.config.core,
- org.eclipse.smarthome.core.service,
- org.osgi.service.cm;version="1.5.0",
+Import-Package: com.google.gson,
+ com.google.gson.annotations,
+ org.apache.commons.io,
+ org.apache.commons.lang,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.core.service,
+ org.osgi.framework;version="1.5.0",
```

Comment:
Please remove the version constraints (unless we have them?)
Also remove it from slf4j while you are at it :-)

Reply:
to be honest, I dont know about this constraint. I was fooled by the constraint on `org.osgi.service.cm;version="1.5.0"` and left it on `org.osgi.framework`, too. Will remove it.